src/controllers/tasks/active.officers.controller.ts

```diff
@@ -45,6 +46,10 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
     const session: Session = req.session as Session;
     const activeOfficerDetailsBtnValue = req.body.activeOfficers;
+
+    if (!isRadioButtonValid(activeOfficerDetailsBtnValue)) {
+      return next(new Error("No valid radio button id in request"));
```

Comment:
See my other comment but I think the log output here and below needs to change slightly as it's the radio button value/choice which is invalid and not the id of the radio button.